packages/block-library/src/navigation/edit/index.js

```diff
@@ -355,7 +355,9 @@ function Navigation( {
 	};
 
 	const onSelectNavigationMenu = ( menuId ) => {
-		handleUpdateMenu( menuId );
+		handleUpdateMenu( menuId, {
+			focusNavigationBlock: true,
+		} );
```

Comment:
When testing I found that - as per the code - this focuses the Nav block in the canvas when switching menus.

I wanted to question this, because surely if you are within the list view and you switch menus you'd expect focus to the remain on the menu switching UI? Or at the very least within the editable List View in the block inspector controls?

This change also _appears_ to be outside the scope of this PR. Maybe I missed something but it feels like we should tackle this aspect separately.

Reply:
You're right that it's outside the scope of the PR but I also think it's inconsistent that we focus on the canvas when we create and on the inspector when we switch. I think in both cases we should probably not move focus, it should be returned tot he toggle of the dropdown. I'll remove this change from here.